plugin/src/main/scala/org/virtuslab/akkasaferserializer/SaferSerializerPlugin.scala

```diff
@@ -0,0 +1,22 @@
+package org.virtuslab.akkasaferserializer
+
+import scala.tools.nsc.Global
+import scala.tools.nsc.plugins.{Plugin, PluginComponent}
+
+class SaferSerializerPlugin(override val global: Global) extends Plugin {
+  override val name: String = "safer-serializer-plugin"
+  override val description: String = "checks for Behavior[A] and ActorRef[A]"
+  private val pluginOptions = new PluginOptions(false)
+
+  private val gatherTypes = new SaferSerializerPluginComponent(pluginOptions, global)
+  override val components: List[PluginComponent] = List(gatherTypes)
+
+  override def init(options: List[String], error: String => Unit): Boolean = {
+    pluginOptions.verbose = options.exists(_.equals("verbose"))
```

Comment:
Nit: `equals` is uncommon in Scala code, just `_ == "verbose"`

Reply:
(likely not a problem here, but `equals` doesn't check for call target being null, while `==` does)